test/functional/wallet_listsinceblock.py

```diff
@@ -5,13 +5,15 @@
 """Test the listsincelast RPC."""
 
 from test_framework.test_framework import BitcoinTestFramework
+from test_framework.messages import BIP125_SEQUENCE_NUMBER
 from test_framework.util import (
     assert_array_result,
     assert_equal,
     assert_raises_rpc_error,
     connect_nodes,
 )
 
+from decimal import Decimal
```

Comment:
nit: IIRC the codebase convention is to place language imports above codebase ones